pyproject.toml

```diff
@@ -70,6 +70,8 @@ include = [
     "/httpx",
     "/CHANGELOG.md",
     "/README.md",
+    "/tests",
+    "/requirements.txt",
```

Comment:
```suggestion
    "/tests",
```

Reply:
Why without `requirements.txt`?  The file is needed to get list of requirements needed to run tests.

Reply:
Just to enforce the `sdist` files that we already have in `httpcore`.

Any other changes should go through the traditional path, which is `discussion -> issue -> pull request`.